examples/src/main/java/com/teamdev/jxbrowser/examples/PrintToPdf.java

```diff
@@ -63,7 +64,8 @@ public static void main(String[] args) {
             });
             tell.proceed(pdfPrinter);
         });
-        browser.navigation().loadUrlAndWait("https://google.com");
+        browser.navigation()
+               .loadUrlAndWait("https://webglsamples.org/dynamic-cubemap/dynamic-cubemap.html");
```

Comment:
For Print Preview example it's better to load a text page. Let's load https://en.wikipedia.org/wiki/Printing